3회차/백동원/1_모음이보이지않는사람.py

```diff
@@ -1,3 +1,15 @@
 import sys
 
 sys.stdin = open("_모음이보이지않는사람.txt")
+T = int(input())
+moum = ['a', 'e', 'i', 'o', 'u']
+for _ in range(1, T + 1):
+    word = list(input())
+    words = []
+    result = ''
+    for a in word:
+        if a not in moum:
+            words.append(a)
+    for b in words:
+        result += b
```

Comment:
좋은 풀이네요 😄 
바로 `result`에 문자를 추가하는 방법도 괜찮을 것 같아요!
```suggestion
    result = ''
    for a in word:
        if a not in moum:
            result += a
```